index.html

```diff
@@ -270,26 +276,26 @@ <h3>Multi-Post Stories</h3>
 
         <!--Contact Forms-->
         <div class="contact flame">
-          <div class="Mname"><h1>CONTACT US</h1></div>
+          <div class="Cname"><h1>CONTACT US</h1></div>
```

Comment:
- [OPTIONAL] I suggest removing the `<h1>` tag from here and using another hx tag (`<h2>`, `<h3>`, etc ...) to Follow best practices for HTML **Only Use H1 Elements Once Per Page for optimal SEO**. for more info, you can check this [Link](https://techstacker.com/seo-only-use-h1-elements-once-per-page/).